ocaml/idl/schematest.ml

```diff
@@ -3,7 +3,7 @@ let hash x = Digest.string x |> Digest.to_hex
 (* BEWARE: if this changes, check that schema has been bumped accordingly in
    ocaml/idl/datamodel_common.ml, usually schema_minor_vsn *)
```

Comment:
![image](https://github.com/xapi-project/xen-api/assets/5189409/49a4b96f-10d9-4e46-89db-3d3fd6f4e2b6)

Reply:
I did..............(though not strictly needed here)? 🤨